tests/conftest.py

```diff
@@ -48,7 +48,7 @@ def icon_grid_simple_path(pytestconfig):
 
 @pytest.fixture
 def icon_filepath_executable() -> str:
-    which_icon = subprocess.run(["which", "icon"], capture_output=True, check=False)
+    which_icon = subprocess.run(["which", "icon"], capture_output=True, check=False)  # noqa: S607
```

Comment:
I think not needed, need to check why you have this issues.
```suggestion
    which_icon = subprocess.run(["which", "icon"], capture_output=True, check=False)
```